fog/view/enclave/impl/src/oblivious_utils.rs

```diff
@@ -589,18 +583,25 @@ mod tests {
 
         assert!(result.is_err());
     }
+
     #[test]
     fn collate_shard_query_responses_different_ciphertext_lengths_returns_correct_client_ciphertexts(
     ) {
         let client_search_keys: Vec<Vec<u8>> = (0..3).map(|num| vec![num; 10]).collect();
-        let ciphertext_values = [28u8, 5u8, 128u8];
+        let ciphertext_values = [5u8, 28u8, 128u8];
```

Comment:
Why the re-order?

Reply:
That change was from another PR that got merged to this branch prematurely, but it makes it easier to test the results.